db/migrate/20180706151804_create_books.rb

```diff
@@ -1,5 +1,6 @@
 class CreateBooks < ActiveRecord::Migration[5.2]
   def change
+    drop_table :books
```

Comment:
This migration will break, if the table `books` does not exist, this will not work.
If you want, we can make a call later